zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -587,23 +598,20 @@ impl<F: Field> SignVerifyChip<F> {
             )?
         };
 
-        let pk_hash_rlc = self.assign_rlc_le(
+        let pk_hash_word = self.assign_word(config, ctx, chips, "pk_hash", pk_hash);
```

Comment:
pk_hash is from `keccak.digest()` which is big endian. So the handling here might in incorrect

Reply:
fixed in [75fa078](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/75fa078f3d271ee039c2a12f993b5c27986cab0a)